### ISSUE TITLE
Disabling selection does not prevent the day from being focusable

### ISSUE THREAD

**Issue by nebulousGirl** [uvarov-frontend/vanilla-calendar#13]
I need your calendar to display specific dates for events, so I chose to disable the selection of a date:
```
 settings: {
    selection: {
      // day: false,
      day: 'single',
    },
  },
```
The problem I have with this is that while I cannot click on the date anymore, tabbing still focuses on not clickable days which should not be the case from an accessibility point of view.

Does a solution exists for this within this plugin, or do I have to add a script on top of it?

**Comment by uvarov-frontend** [comment]
Hi, thanks for reporting the issue! Indeed, from an accessibility point of view, disabled elements should not be focused. For each disabled button, you need to add `tabindex="-1"` as a tag attribute. Right now this feature is not in the plugin, but I will definitely add it in the next versions.
You can add a CSS rule for disabled dates `outline: none;`, but I don't think that's what you need...

**Comment by uvarov-frontend** [comment]
Issue fixed in version 2.0.0
